packages/asc-ui/src/utils/themeUtils.ts

```diff
@@ -137,12 +137,12 @@ export const focusStyleOutline = (width?: number, offset?: number) => ({
 }: {
   theme: Theme.ThemeInterface
 }) => css`
-  &&:focus {
+  &:focus {
```

Comment:
should be &&:focus: this will make sure it will override a hover state with outlines. I introduced this when resolving this issue: #131

Reply:
Thanks, I'll add a comment there.